webapp/src/selectors/index.tsx

```diff
@@ -27,3 +27,11 @@ export const getRhsState = (state: any): {isSidebarOpen: boolean} => {
 export const getSubscribeModalState = (state: any): SubscribeModalState => {
     return state[pluginPrefix].openSubscribeModalSlice;
 };
+
+export const getApiRequestCompletionState = (state: any): ApiRequestCompletionState => {
+    return state[pluginPrefix].apiRequestCompletionSlice;
+};
+
+export const getApiQueriesState = (state: any): ApiQueriesState => {
+    return state[pluginPrefix].azureDevopsPluginApi?.queries;
```

Comment:
```suggestion
    return state[pluginPrefix].azureDevopsPluginApi.queries;
```
Is there really a need for optional chaining here?

Reply:
yes this is async so it may be undefined initially